package.json

```diff
@@ -8,6 +8,7 @@
     "dev": "turbo run dev",
     "generate": "turbo run generate --token=${TURBO_TOKEN}",
     "lint": "turbo run lint --token=${TURBO_TOKEN}",
+    "test:system": "turbo run test:system --token=${TURBO_TOKEN}",
     "test:visual": "turbo run test:visual",
```

Comment:
Does this line also need the turbo token?

Reply:
I can see why we might not want it, but just thought I'd check

Reply:
Yep, this is for remote caching. We should be able to rely on the cache if the code hasn't changed.